apps/fabric-website-resources/src/components/pages/ColorsPage.tsx

```diff
@@ -444,14 +438,14 @@ export class ColorsPage extends BaseComponent<{}, IColorsPageState> {
   private _baseColorSlotPicker = (baseSlot: BaseSlots, title: string): JSX.Element => {
     let colorChangeTimeout: number;
 
-    function _onColorChanged(newColor: string): void {
+    const onChange = (ev: any, newColor: IColor): void => {
```

Comment:
Are we unable to type these `ev` parameters? I thought they might be `React.MouseEvent<HTMLElement>` glancing at the code.

Reply:
That was more laziness than anything else...I can add a type.

Reply:
No worries, just figured might as well. Lmk if you want me to take a pass at it. Great work btw!